src/framework/theme/components/chat/chat-message.component.ts

```diff
@@ -193,15 +202,34 @@ export class NbChatMessageComponent {
    */
   @Input() type: string;
 
+  @Input() customMessageData: any;
+
+  private readonly defaultMessageTypes: string[] = ['text', 'file', 'map', 'quote', undefined];
+
   constructor(protected domSanitizer: DomSanitizer) { }
 
   getInitials(): string {
     if (this.sender) {
       const names = this.sender.split(' ');
-
       return names.map(n => n.charAt(0)).splice(0, 2).join('').toUpperCase();
     }
-
     return '';
   }
+
+  isDefaultMessageType(msgType: string): boolean {
+    return this.defaultMessageTypes.some(el => el === msgType);
```

Comment:
`this.defaultMessageTypes.includes` would be simpler as it doesn't require comparison callback